commands/dkcheck.ml

```diff
@@ -97,43 +104,42 @@ let _ =
     ; ( "-q"
       , Arg.Unit (fun () -> Env.set_debug_mode "q")
       , " Quiet mode (equivalent to -d 'q')" )
-    ; ( "-e"
-      , Arg.Set export
-      , " Generates an object file (\".dko\")" )
-    ; ( "-nc"
+    ; ( "--no-color"
       , Arg.Clear Errors.color
       , " Disables colors in the output" )
-    ; ( "-stdin"
+    ; ( "-nc"
```

Comment:
This option name is long (i.e. more than one char) so it should have double dash, no ?

Reply:
This flag is not documented anyway (no entry in the `--help` message).
We do have a documented version of it, `--no-color`, which should be satisfying.
`-nc` and `-nl` are silently still supported to prevent all scripts relying on them to suddenly fail.